input/definitions/MeasureReport/StructureDefinition-criteriaReference.json

```diff
@@ -0,0 +1,88 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "criteriaReference",
+  "url": "http://hl7.org/fhir/StructureDefinition/criteriaReference",
+  "version": "1.0.0",
+  "name": "CriteriaReferenceExtension",
+  "title": "Criteria Reference Extension",
+  "status": "active",
+  "experimental": false,
+  "date": "2023-12-15",
+  "publisher": "HL7 International / Clinical Decision Support",
+  "contact": [
+    {
+      "telecom": [
+        {
+          "system": "url",
+          "value": "http://www.hl7.org/Special/committees/dss"
+        }
+      ]
+    }
+  ],
+  "description": "Specifies which criteria is the input for calculations.",
+  "jurisdiction": [
+    {
+      "coding": [
+        {
+          "system": "urn:iso:std:iso:3166",
+          "code": "US"
+        }
+      ]
+    }
+  ],
+  "purpose": "The definition must be able to specify which criteria is the input used.",
+  "fhirVersion": "4.0.1",
+  "baseDefinition": "http://hl7.org/fhir/StructureDefinition/Extension",
+  "derivation": "constraint",
+  "mapping": [
+    {
+      "identity": "rim",
+      "uri": "http://hl7.org/v3",
+      "name": "RIM Mapping"
+    }
+  ],
+  "kind": "complex-type",
+  "abstract": false,
+  "context": [
+    {
+      "type": "element",
+      "expression": "MeasureReport.supplementalData"
+    },
+    {
+      "type": "element",
+      "expression": "MeasureReport.evaluatedResource"
+    }
+  ],
+  "type": "Extension",
+  "differential": {
+    "element": [
+      {
+        "id": "Extension",
+        "path": "Extension",
+        "short": "Criteria reference",
+        "definition": "Reference a criteria by specifying that criteria's unique id",
+        "min": 0,
+        "max": "*"
+      },
+      {
+        "id": "Extension.url",
+        "path": "Extension.url",
+        "type": [
+          {
+            "code": "uri"
+          }
+        ],
+        "fixedUri": "http://hl7.org/fhir/StructureDefinition/extension-criteriaReference"
```

Comment:
I've suggested a change to the url above, but even without that, this is incorrect, the fixedUri constraint here must match the url element of the StructureDefinition overall. If my change above is accepted, then that would be:
http://hl7.org/fhir/StructureDefinition/measurereport-criteriaReference

Reply:
```suggestion
        "fixedUri": "http://hl7.org/fhir/StructureDefinition/cqf-criteriaReference"
```